.gitmodules

```diff
@@ -9,3 +9,6 @@
 [submodule "src/abi"]
 	path = src/abi
 	url = ../../twizzler-operating-system/abi.git
+[submodule "src/bin/src/bin/test-tiny-http/src/tiny-http-twizzler"]
+	path = src/bin/src/bin/test-tiny-http/src/tiny-http-twizzler
+	url = git@github.com:ClariseTG/tiny-http-twizzler.git
```

Comment:
We'll need to update this to twizzler-operating-system/tiny-http-twizzler. I just moved the repo, so it should work.